src/report/report_generate.py

```diff
@@ -0,0 +1,106 @@
+"""
+Script generating monthly report
+"""
+
+import report_utils as utils
+import sys
+from report_data import get_report_data
+from docx import Document
+from docx.enum import section, text
+from os import remove
+
+
```

Comment:
why the code isn't in a function?

Reply:
I'm going to change it to be a function in this sprint

Reply:
This should have been done in a previous sprint. Update changes to this pull request.